app.js

```diff
@@ -0,0 +1,63 @@
+const express = require('express');
+const Sequelize = require('sequelize');
```

Comment:
If you keep adding everything to this file, you will end up with a few thousand lines of code here. I don't think you want this to happen :)
Let's rename this file `index.js` and keep here only the initial content, from the previous PR.
Then create a folder `app` and `app/models`. In `modes` you will add all the models, a separate file for each one, in this case `app/models/user.js`.